### PR TITLE
Add hooks refactor to my react app

### PR DESCRIPTION
**Pull request by edwardoluyara**
This was built to set up hooks refactor for my react projects

- Add `btnName.js` to `components` folder with a variable containing all the calculator numbers
- Add the `useState` keyword to import react in `calculator.js` in the `components` folder.
- Create `btnClass` variable that contains the calculator output.
- Return `btnClass` to append the number.

### CODE REVIEW

File: src/components/calculator.js [hunk 1]
```diff
@@ -1,44 +1,34 @@
...
-        <button type="button" onClick={this.handleClick} className="gray-like-btn">6</button>
-        <button type="button" onClick={this.handleClick} className="orange-like-btn">-</button>
-        <button type="button" onClick={this.handleClick} className="gray-like-btn">1</button>
-        <button type="button" onClick={this.handleClick} className="gray-like-btn">2</button>
-        <button type="button" onClick={this.handleClick} className="gray-like-btn">3</button>
-        <button type="button" onClick={this.handleClick} className="orange-like-btn">+</button>
-        <button type="button" onClick={this.handleClick} className="gray-like-btn btn-zero">0</button>
-        <button type="button" onClick={this.handleClick} className="gray-like-btn">.</button>
-        <button type="button" onClick={this.handleClick} className="orange-like-btn">=</button>
-      </div>
```
Comment (hiromataba) [inline src/components/calculator.js:39]: - [ ] Please fix your design according to the project guideline, Your equal `=` sign is not at the right position
- Your design :point_down: 

image

- Project design :point_down: 

image

**Review by hiromataba — CHANGES_REQUESTED** [review]
Hi :wave: @oluyaratosin123 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
### To highlight :+1: :tada: 
- No linter errors :heavy_check_mark: 
- Nice PR title and description :+1: 
- Your app is  working correctly :heavy_check_mark: 

## Required Changes ♻️

- [ ] Please update your README file with the commands on how to install npm `npm install`, how to run the project in the browser `npm start`, and how to run tests `npm test`

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/calculator.js [hunk 2]
```diff
@@ -1,44 +1,34 @@
-import React from 'react';
+import React, { useState } from 'react';
 import calculate from '../logic/calculate';
+import btnValues from './btnNames';
+import '../App.css';
 
-class Calculator extends React.Component {
```
Comment (SajjadAhmad14) [inline src/components/calculator.js:4]: Please fix the design issue as it was pointed out by the previous code reviewer.
design

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Edward, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by edwardoluyara** [comment]
Hello @SajjadAhmad14,  Do you have any idea on how I could do this pls?

**Review by Gambit142 — APPROVED** [review]
Hi @oluyaratosin123 👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

Congratulations! 🎉 

### Optional

It would be nice to make the _**mathematical operator symbols**_  apart from the `=` sign to be visible on the display screen when clicked. That would improve the User Experience (UX) of anybody using your calculator app.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**